deploy/crds/crd-certificates.yaml

```diff
@@ -170,6 +170,9 @@ spec:
                             name:
                               description: 'Name of the resource being referred to. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
                               type: string
+                literalSubject:
+                  description: LiteralSubject is an LDAP formatted string that represents the X.509 Subject field   https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.6 Use this *instead* of the Subject field if you need to ensure the correct ordering of the RDN sequence, such as when issuing certs for LDAP authentication. See https://github.com/cert-manager/cert-manager/issues/3203, https://github.com/cert-manager/cert-manager/issues/4424. This field is alpha level and is only supported by cert-manager installations where LiteralCertificateSubject feature gate is enabled on both cert-manager controller and webhook.
```

Comment:
Thanks for the spot. This looks like this because the multiple lines from the types_certificate.go definition are concatenated in the crd yaml. I will implement your proposal regardless.